internal/util/apiclient/websocket_log_reader.go

```diff
@@ -18,7 +18,7 @@ import (
 
 var workspaceLogsStarted bool
 
-func ReadWorkspaceLogs(ctx context.Context, activeProfile config.Profile, workspaceId string, projectNames []string, follow, showWorkspaceLogs bool) {
+func ReadWorkspaceLogs(ctx context.Context, activeProfile config.Profile, workspaceId string, projectNames []string, follow, showWorkspaceLogs bool, timeNow time.Time) {
```

Comment:
Let's rather have `from *time.Time` so you can check if it's set with `!= nil` and functions that need all logs can pass nil instead of `time.Time{}` which is more intuitive.

Reply:
That's true 💯

Reply:
@unsuman Let's call the argument `from` instead of `timeNow`. `timeNow` is ambiguous.